c/tskit/tables.c

```diff
@@ -578,14 +584,46 @@ tsk_individual_table_get_row(
     return ret;
 }
 
+int
+tsk_individual_table_set_metadata_schema(tsk_individual_table_t *self,
+    const char *metadata_schema, tsk_size_t metadata_schema_length)
+{
+    int ret = 0;
```

Comment:
I usually put a blank line between declarations and code.

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/493/commits/01d615852dba1b64d5a463fccb869bf6338088fa